examples/applications/invoke-command/src/main.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+mod chariott_provider;
+
+use std::sync::Arc;
+
+use chariott_common::error::Error;
+use chariott_common::shutdown::RouterExt as _;
+use chariott_proto::{
+    provider::provider_service_server::ProviderServiceServer,
+    runtime::{
+        intent_registration::Intent, intent_service_registration::ExecutionLocality,
+    },
+};
+use examples_common::chariott;
+use tonic::transport::Server;
+
+use crate::chariott_provider::ChariottProvider;
+
+chariott::provider::main!(wain);
+
+async fn wain() -> Result<(), Error> {
+    let (url, socket_address) = chariott::provider::register(
+        "sdv.invoke.controller",
+        "0.0.1",
+        "sdv.invoke.controller",
+        [Intent::Invoke, Intent::Discover],
```

Comment:
Consider putting these in the order of the enum: https://github.com/eclipse/chariott/blob/b0d2add377f1740f26c52bfb4e234ce415b91b81/src/chariott_grpc.rs#:~:text=const%20INTENT_MAPPING_DISCOVER%3A,%3D%205%3B

Reply:
updated